File: python/guessGame.py
import random

print("\n*----------Welcome to Guess Number Game----------*\n")

n = random.randint(1, 100)
count = 0

while True:
    guess = int(input("Enter your guess: "))
    count += 1

    if n < guess:
        print("Lower")
    elif n > guess:
        print("Higher")
    else:
        break

print(f"\n Congratulations! You guessed it in {count} attempts")

File: java/guessGame.java
import java.util.*;

public class guessGame {

    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        Random rand = new Random();
        int count = 0;
        int n = rand.nextInt(100) + 1;
        int guess;
        do {
            System.out.print("\nEnter your guess\n");
            guess = sc.nextInt();
            count++;

            if (n < guess) {
                System.out.print("\nLower");
            } else if (n > guess) {
                System.out.print("\nHigher");
            }
        } while (guess != n);
        System.out.println(
            "\nCongratulations you guess it right in " + count + " attempt\n"
        );
    }
}
